src/mapblock_mesh.cpp

```diff
@@ -290,7 +290,9 @@ static u16 getSmoothLightCombined(v3s16 p, MeshMakeData *data)
 	{
 		//table of precalculated gamma space multiply factors
 		//light^2.2 * factor (0.75, 0.5, 0.25, 0.0), so table holds factor ^ (1 / 2.2)
-		static const float light_amount[4] = { 0.877424315, 0.729740053, 0.532520545, 0.0 };
+		static const float light_amount[4] = { g_settings->getFloat("ambient_occlusion_strength"), 
+				g_settings->getFloat("ambient_occlusion_strength")-0.1f, 
+				g_settings->getFloat("ambient_occlusion_strength")-0.3f, 0.0 };
```

Comment:
@TriBlade9 Ah, yes, it is only done once.  The calls should still be de-duplicated though.  No sense wasting CPU time even on init, and it will be more readable.